modules/ROOT/pages/migration-module-vm.adoc

```diff
@@ -84,7 +84,7 @@ Main differences are:
 
 * The listener points to a config and can only listen to a queue defined in that config.
 * Now, you can use the `<vm:response>` element to control the response. If not provided, the message payload that results at the end of the flow will be settings.
-* You no longer need to use an `exchange-pattern` to control whether or not a response is sent. The connector automatically knows when to send it depending on the message being generated through the `<vm:publish>` or `<vm:publish-response>` operations.
+* You no longer need to use an `exchange-pattern` to control whether or not a response is sent. The connector automatically knows when to send it depending on the message being generated through the `<vm:publish>` or `<vm:publish-consume>` operations.
```

Comment:
Suggested wording: "The connector automatically knows when to send the response, depending on the message generated by the `<vm:publish>` or `<vm:publish-consume>` operations."